src/Cake.Common/Tools/NuGet/Pack/NuGetPacker.cs

```diff
@@ -214,6 +214,12 @@ private ProcessArgumentBuilder GetArguments(FilePath filePath, NuGetPackSettings
                     settings.Properties.Select(property => string.Concat(property.Key, "=", property.Value))));
             }
 
+            // Keep temporarily autogenerated NuSpec file?
+            if (settings.KeepTemporaryNuSpecFile)
+            {
+                builder.Append("-KeepTemporaryNuSpecFile");
```

Comment:
@asbjornu ok, you lost me. What does that property have to do with this PR?  Bear in mind, any additions to the arguments builder will ultimately be passed through to the underlying tool, in this case NuGet.exe, and I don't think that `-KeepTemporaryNuSpecFile` is a command line argument for NuGet.exe, is it?  Therefore, the settings that you are passing in via the `NuGetPackSettings` is only used prior to calling NuGet.exe.  Or have I missed something?